Baby MonitorTests/Media/SoundDetectionServiceTests.swift

```diff
@@ -0,0 +1,171 @@
+//
+//  SoundDetectionService.swift
+//  Baby MonitorTests
+
+import Foundation
+import XCTest
+import RxSwift
+import RxTest
+import AVFoundation
+@testable import BabyMonitor
+
+class SoundDetectionServiceTests: XCTestCase {
+
+    func testShouldStartCapturing() {
+        // Given
+        let microphoneServiceMock = AudioMicrophoneServiceMock()
+        let noiseDetectionServiceMock = NoiseDetectionServiceMock()
+        let cryingDetectionServiceMock = CryingDetectionServiceMock()
+        let cryingEventsServiceMock = CryingEventsServiceMock()
+        let sut = SoundDetectionService(microphoneService: microphoneServiceMock,
+                                        noiseDetectionService: noiseDetectionServiceMock,
+                                        cryingDetectionService: cryingDetectionServiceMock,
+                                        cryingEventService: cryingEventsServiceMock)
```

Comment:
Please consider refactoring these into class level properties set in `setUp()` to slim down those test `func`s